database/src/main/kotlin/no/nav/su/se/bakover/database/kontrollsamtale/KontrollsamtaleRepo.kt

```diff
@@ -0,0 +1,32 @@
+package no.nav.su.se.bakover.database.kontrollsamtale
+
+import no.nav.su.se.bakover.database.PostgresSessionFactory
+import no.nav.su.se.bakover.database.insert
+import no.nav.su.se.bakover.domain.kontrollsamtale.Kontrollsamtale
+
+interface KontrollsamtaleRepo {
```

Comment:
Vi har flyttet alle interface *Repo til domenet nå :)

Reply:
Er det noen grunn til det? :P

Reply:
Knyttet til DDD - domenet(business logikken) skal eie alt som er knyttet til det (service inngår som en del av domenet). Der eier domenet interfacene som client og database implementerer. Dette for å få et tydelig skille.

Reply:
Men dette interfacet er det bare database som implementerer. Så da blir det vel litt rart å skulle skille det fra database?